samples/trace/package.json

```diff
@@ -11,9 +11,10 @@
   },
   "repository": "GoogleCloudPlatform/opentelemetry-operations-js",
   "dependencies": {
-    "@google-cloud/opentelemetry-cloud-trace-exporter": "^0.5.0",
+    "@google-cloud/opentelemetry-cloud-trace-exporter": "^0.6.0",
     "@opentelemetry/api": "^0.12.0",
     "@opentelemetry/node": "^0.12.0",
     "@opentelemetry/tracing": "^0.12.0"
-  }
+  },
+  "version": "0.6.0"
```

Comment:
Could you please move this after name, just to be consistent with other package.json